crates/assets/Cargo.toml

```diff
@@ -9,7 +9,7 @@ version = "1.7.2"
 [dependencies]
 # External dependencies
 base58 = {version = "0.2.0", default-features = false}
-hex = {version = "0.4.3", default-features = false}
+hex = {version = "0.4.3", default-features = false, features = ["alloc"]}
```

Comment:
We need the alloc feature?

Reply:
let me guess, the feature was enabled by the did pallet. Since the dependency was removed there, the feature was also not enabled?

Reply:
Yes